roadmap/implementers-guide/src/node/README.md

```diff
@@ -11,3 +11,14 @@ The architecture of the node-side behavior aims to embody the Rust principles of
 Many operations that need to be carried out involve the network, which is asynchronous. This asynchrony affects all core subsystems that rely on the network as well. The approach of hierarchical state machines is well-suited to this kind of environment.
 
 We introduce a hierarchy of state machines consisting of an overseer supervising subsystems, where Subsystems can contain their own internal hierarchy of jobs. This is elaborated on in the next section on Subsystems.
+
+## Assumptions
+
+The Node-side code comes with a set of assumptions that we build upon. These assumptions encompass most of the basic blockchain functionality that we build upon.
+
+We assume these things about the underlying node environment:
```

Comment:
It parses weirdly to say, in effect: "The node-side code... assumes these things about the underlying node". 

I suspect that "node environment" is intended to bind more tightly, but that wasn't my first reading.

```suggestion
We assume these things about the underlying environment:
```

Reply:
Actually, I like @drahnr 's suggestion for this line better.